extensions/chocolatey-core.extension/extensions/Remove-Process.ps1

```diff
@@ -0,0 +1,103 @@
+<#
+.SYNOPSIS
+    Ensure that process is stopped in reliable way
+
+.DESCRIPTION
+    Close the processes matching filters gracefully first, then forcefully.
+    If no process is found, function will simply do and return nothing.
+
+.EXAMPLE
+    notepad; Remove-Process notepad -PathFilter 'system32'
+    
+    Close main window of notepad that has 'system32' word in its path
+
+.EXAMPLE
+    Remove-Process notepad -WaitFor 30
+    notepad; notepad  #in another shell 
+    
+    Close all instances of notepad but wait for them up to 30 seconds to start
+
+.OUTPUTS
+    Array of closeed processes with details about each one.
+
+.NOTES
+    https://github.com/chocolatey-community/chocolatey-coreteampackages/issues/1364 
+#>
+
+function Remove-Process {
```

Comment:
Calling this function `Remove-Process` is incorrect.
This would imply that the process itself is removed, but in truth it is just being stopped.

Reply:
Whaaaa ?
Its removed from the process list. Also, `Stop-Process` is taken already (`kill`) and you must choose from approved verbs.

Reply:
I first used `Ensure-NoProcess` but there were warnings every time on installation of any package using extension.